tests/cupy_tests/core_tests/test_raw.py

```
@@ -11,9 +11,10 @@
 
 _test_source1 = r'''
 extern "C" __global__
-void test_sum(const float* x1, const float* x2, float* y) {
+void test_sum(const float* x1, const float* x2, float* y, unsigned int N) {
     int tid = blockDim.x * blockIdx.x + threadIdx.x;
-    y[tid] = x1[tid] + x2[tid];
+    if (tid < N)
+        y[tid] = x1[tid] + x2[tid];
```

This fixes a bug silently ignored in by the driver API...